### ISSUE TITLE
Bookmarks in pdf export

### ISSUE THREAD

**Issue by suppenkaspar109** [peerdavid/remapy#49]
Hello all,
first of all thank you very much for the great program.
Is there a possibility to export the exisiting bookmarks with the pdf export of a document. 
Right now, all bookmarks are lost.
This would be very helpful especially for books. Many thanks

**Comment by peerdavid** [comment]
Hello,

thanks for your input - what I wonder is if you open the raw document (right click -> open raw) are bookmarks included i.e. are bookmarks only missing in the annotated document?

Thank you very much,
David

**Comment by suppenkaspar109** [comment]
Exactly, this is where the problem exists. Everything fits with the raw data, but the bookmarks are missing in the annotated PDFs.

**Comment by peerdavid** [comment]
Thank you very much, I will check if I can keep bookmarks during the rendering process...
